src/webhooks/options-automator/options-automator.test.ts

```diff
@@ -279,16 +279,42 @@ describe('options-automator webhook', function () {
 
   describe('sendOptionAutomatorUpdatesToDataDog tests', function () {
     it('should send the right payload', async function () {
-      sendOptionAutomatorUpdatesToDataDog(testpayload, 1699563828);
-      expect(datadogApiInstanceSpy).toHaveBeenCalledWith({
+      await sendOptionAutomatorUpdatesToDataDog(testparitalpayload, 1699563828);
```

Comment:
nit: testPartialPayload

can we use camelcase here for our variables for test payloads?